js/privatebin.js

```diff
@@ -362,6 +361,36 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             }
         };
 
+        /**
+         * converts size in bytes to human readable string
+         *
+         * The string is expected to be optional digits, followed by a time.
+         * Supported times are: min, hour, day, month, year, never
+         * Examples: 5min, 13hour, never
+         *
+         * @name Helper.humanizeBytes
+         * @function
+         * @param  {number} bytes
+         * @param {decimals} decimals - optional
+         * @return {String}
+         */
+        me.humanizeBytes = function(bytes, decimals = 2) {
+            const sizes = [
+                I18n._('Bytes'), I18n._('KB'), I18n._('MB'), I18n._('GB'),
+                I18n._('TB'), I18n._('PB'), I18n._('EB'), I18n._('ZB'), I18n._('YB')
```

Comment:
Please use the [power of 1024 based binary prefixes](https://en.wikipedia.org/wiki/Binary_prefix) to avoid any confusion with the power of 1000 based SI unit prefixes. The former ones already got translated for those languages that do use a different unit letter - for example the "o" for "octet" in French:
https://github.com/PrivateBin/PrivateBin/blob/33bcce593a3e8baa7ebdf6abf00175b399582fc9/i18n/fr.json#L123-L131